### DISCUSSION TITLE
is there a way to expand GBA roms so it uses the full screen and turn off retroarch notifications?

### DISCUSSION THREAD

**Discussion by viola8757** [OnionUI/Onion#504] [Q&A] [answered]
I think in the stock os i had it on full screen. But now that i've updated it to onion os, there is a bar at the top and bottom of screen where i see the executed RetroArch commands like save state notifications. I would like those notifications to either overlay on the game or just turn it off.

**Comment by Znidz** [comment, marked as answer]
You can set this in the Retroarch settings. 
I think generally people prefer an unstretched image. I'm assuming this is deliberate.    
Select+Home will open the settings. Then B to go up a level. 
Then settings>scaling.      
You can change the retroarch notifications at the same level under "on screen notifications".

**Comment by Sichroteph** [comment]
Don't forget to save this setting by going to override - save core override

**Comment by Drewby-b** [comment]


**Reply by Znidz** [reply]
I get this issue as well. I don't know how to fix.

**Reply by Drewby-b** [reply]
> I get this issue as well. I don't know how to fix.

The fix is to wait for a real replacement to stock. If we can't do something so simple as change to full screen and save the setting then Onion OS isn't it.
